integ-test/src/test/java/org/opensearch/sql/legacy/MethodQueryIT.java

```diff
@@ -38,7 +38,7 @@ public void queryTest() throws IOException {
         "select address from %s where query('address:880 Holmes Lane') limit 3",
         TestsConstants.TEST_INDEX_ACCOUNT));
     Assert.assertThat(result,
-        containsString("query_string\":{\"query\":\"address:880 Holmes Lane"));
+        containsString("query_string\\\":{\\\"query\\\":\\\"address:880 Holmes Lane"));
```

Comment:
this wasn't working?

Reply:
There were some issues with escaped characters causing it to fail.